src/fidesctl/core/sync.py

```diff
@@ -0,0 +1,51 @@
+"""This module handles the logic for syncing remote resource versions into their local file."""
+from typing import Dict
+
+import yaml
+from fideslang.manifests import load_yaml_into_dict
+
+from fidesctl.cli.utils import echo_green, print_divider
+from fidesctl.core.api_helpers import get_server_resource
+from fidesctl.core.utils import get_manifest_list
+
+
+def sync(manifests_dir: str, url: str, headers: Dict[str, str]) -> None:
+    """
+    If a resource in a local file has a matching resource on the server,
+    write out the server version into the local file.
+    """
+
+    manifest_path_list = get_manifest_list(manifests_dir)
+
+    print_divider()
+    for manifest_path in manifest_path_list:
+        print(f"Syncing file: '{manifest_path}'...")
+        manifest = load_yaml_into_dict(manifest_path)
+        updated_manifest = {}
+
+        for resource_type in manifest.keys():
```

Comment:
When first running this before looking too closely, I applied the `demo_resources/` and then ran the `sync` command thinking they would be written to the `.fides/` directory (not realizing at first what a pain that would be to map resources to files etc.) Is there a plan in your mind to write all server resources out to default files or should this always be restricted to fides_keys already defined in a manifest?

Reply:
Neville brought this up as well. I specifically designed it to _not_ do that, but its very possible we could add an `--all` flag that loads everything regardless of what exists locally. I'm hesitant to make this the default behavior though